src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
```

Comment:
It's recommended to avoid passing raw values directly into the constructor. Instead, create variables for the radius and color before passing them to the superclass constructor for better readability.